src/test/java/org/alfresco/repo/tagging/TaggingServiceImplTest.java

```diff
@@ -2426,5 +2435,89 @@ public Void execute() throws Throwable
             nodeRefPropInterceptor.setFilterOnGet(true);
         }
     }
+    
+    /**
+     * ALF-21875
+     */
+    @Test
+    public void testTagScopeALF_21875() {
+        
+        TagScopePropertyMethodInterceptor.setEnabled(Boolean.TRUE);
+        AuthenticationUtil.setFullyAuthenticatedUser(AuthenticationUtil.getAdminUserName());
+        String siteName = GUID.generate();
+
+        this.transactionService.getRetryingTransactionHelper().doInTransaction(new RetryingTransactionCallback<Void>() {

```

Comment:
Please keep Alfresco syntax formation we do { in the next line. For the others as well.